client/source/components/RR/RatingBreakdown/RatingBreakdown.jsx

```diff
@@ -20,35 +22,39 @@ const RatingBreakdown = function RatingsBreakdown({
   const filters = helpers.filtered(ratingsFilter);
 
   return (
-    <div className="rating-breakdown">
-      <div className="average-rating">{averageRating}</div>
-      <div className="star-rating">stars here</div>
-      <div className="recommended-percentage">
-        {`${votes}% of reviews recommend this product`}
+    <RatingBreakdownContainer>
+      <div className="rating-breakdown">
+        <div className="rating-details">
+          <div className="average-rating">{averageRating}</div>
+          <div className="star-rating">stars here</div>
```

Comment:
this line can now be replaced with `<AverageStars avgRating={averageRating} />` !!